src/python/grpcio/commands.py

```diff
@@ -157,30 +157,47 @@ def run(self):
     if not self.grpc_python_plugin_command:
       raise CommandError('could not find grpc_python_plugin '
                          '(protoc plugin for GRPC Python)')
+
+    proto_stem = os.path.join(GRPC_STEM, 'src', 'proto')
```

Comment:
Does this value have to be computed in this function or can it be a module-scope constant?